commands/commands.go

```diff
@@ -56,11 +56,20 @@ func GetCommands() []*clitool.Command {
 		{
 			Name:    "scan-pull-requests",
 			Aliases: []string{"sprs"},
-			Usage:   "Scans all the open pull requests in the repo with JFrog Xray for security vulnerabilities.",
+			Usage:   "Scans all the open pull requests in single or multiple repositories with JFrog Xray for security vulnerabilities",
```

Comment:
...pull requests in single or multiple repositories,,, --> ...pull requests within a single or multiple repositories...

Reply:
done